<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWidget</class>
 <widget class="QWidget" name="ClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ClientWidget</string>
  </property>
  <widget class="QGroupBox" name="groupBoxConnexion">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>561</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Connexion au serveur</string>
   </property>
   <widget class="QTextEdit" name="textEditAffichageEtat">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>251</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelEtat">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat de la connexion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAdresseIPServeur">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNumeroPort">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelNumeroPortServeur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Numero de port du serveur :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAdresseIPServeur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse IP du serveur :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonQuitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnexion">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>connexion</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxInformations">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>561</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Informations Client</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>551</width>
      <height>122</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonOrdinateuDistant">
         <property name="text">
          <string>Nom de l'ordinateur distant</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonNomUtilisateur">
         <property name="text">
          <string>Nom de l'utilisateur</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonArchitecture">
         <property name="text">
          <string>Architecture processeur</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonOS">
         <property name="text">
          <string>OS  de l'ordinateur</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEditDistant">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditNomUtilisateur">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditArchitecture">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditOS">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
